src/providers/EtherspotContextProvider.tsx

```diff
@@ -51,6 +53,12 @@ export interface ITotalWorthPerAddress {
   [address: string]: number
 }
 
+export interface IBalanceByChain {
```

Comment:
Your type says `byChain` and param says `perChain`, please match

Reply:
Okh will use `smartWalletBalanceByChain` for all